plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/provider/NamespacedPropertyEditorDialog.java

```diff
@@ -454,6 +912,355 @@ public void handleEvent(Event event) {
+                xPathTextField.setText(XSLTXPathHelper.calculateXPathToNode(treeItemData.getDomNode()));
+            }
+        });
+
+        treeViewWidget.addListener(SWT.Expand, new Listener() {
+            public void handleEvent(Event event) {
+                TreeItem currentItem = (TreeItem) event.item;
+                TreeItemData treeItemData = (TreeItemData) currentItem.getData(TREE_ITEM_DATA_KEY);
+
+                // Explore if not already explored.
+                if (!treeItemData.isMarkedAsExplored()) {
+                    // Make sure to remove the dummy child item.
+                    currentItem.removeAll();
+
+                    Node node = treeItemData.getDomNode();
+
+                    // Attributes.
+                    if (node.hasAttributes()) {
+                        NamedNodeMap attributesMap = node.getAttributes();
+                        for (int i = 0; i < attributesMap.getLength(); i++) {
+                            Node childNode = attributesMap.item(i);
+                            addTreeItem(currentItem, childNode);
+                        }
+                    }
+
+                    // Children.
+                    if (node.hasChildNodes()) {
+                        NodeList childNodes = node.getChildNodes();
+                        for (int i = 0; i < childNodes.getLength(); i++) {
+                            Node childNode = childNodes.item(i);
+                            if (childNode.getNodeType() == Node.ELEMENT_NODE) {
+                                addTreeItem(currentItem, childNode);
+                            }
+                        }
+                    }
+
+                    // Done exploring.
+                    treeItemData.setMarkedAsExplored();
+                }
+            }
+        });
+
+        treeViewWidget.addMouseListener(new MouseListener() {
+            
+            @Override
+            public void mouseDoubleClick(MouseEvent e) {
+                TreeItem[] selection = treeViewWidget.getSelection();
+                if (selection.length > 0) {
+                    Node selectedNode = ((TreeItemData) selection[0].getData(TREE_ITEM_DATA_KEY)).getDomNode();
+                    setSelectedXpath(XSLTXPathHelper.calculateXPathToNode(selectedNode));
+                    dialogShell.dispose();
+                }
+            }
+            
+            // The following two methods were left empty intentionally since they must be overridden
+            @Override
+            public void mouseUp(MouseEvent e) {}
+
+            @Override
+            public void mouseDown(MouseEvent e) {}
+
+        });
+        
+        inlineRadioButton.addSelectionListener(new SelectionAdapter()  {
+            
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                Button source = (Button) e.getSource();
+                if(source.getSelection())  {
```

Comment:
Fix formatting issues #1074 and #1097

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804